docker-compose.yml

```diff
@@ -18,11 +44,22 @@ services:
       HTTPS_CERT_FULLCHAIN: /etc/ssl/fullchain.pem
       HTTPS_CERT_PRIVKEY: /etc/ssl/privkey.pem
       INTERACTIVE: "false"
+      MEDIASOUP_MIN_PORT: 40000
+      MEDIASOUP_MAX_PORT: 40050
+      MEDIASOUP_ANNOUNCED_IP: 10.20.30.12
     ports:
       - "4443:4443"
+      # This should be the same as MEDIASOUP_MIN_PORT and MEDIASOUP_MAX_PORT
+      # You might neeed to increase this number depending on how many client you are connecting.
+      # (Keep the ports range small to improve container statup time)
+      - "40000-40050:40000-40050"
```

Comment:
Do these ports need to be exposed to the host OS? If so, does the user need to accept proxy certificates in the browser for them?

Reply:
This are used for Coturn <-> Dialog communication but again this is the only way I found to communicate two containers. Open to hear alternatives.

Reply:
The _ports_ config shouldn’t affect communication between two services on the same network. They are able to communicate directly without the intervention of the host OS.

* https://docs.docker.com/compose/networking/
* https://docs.docker.com/compose/compose-file/compose-file-v3/#ports